### DISCUSSION TITLE
How run app (ASURE_CONECTION_STRING)

### DISCUSSION THREAD

**Discussion by Tabernol** [ita-social-projects/GreenCity#6644] [General]
Hello, I want to take part in the development of a project to gain experience.
Trying to run the program. I created a database locally, licibfsk created tables. But I get an error at startup because I don't have a value for AZURE_CONNECTION_STRING.
Can you tell me where to get it?
Do I need to create my own Azure Blob Storage?

**Comment by ViktoriyaRyazhska** [comment]
Hi,
yes, you have to create your own

From: MaxKrasnoloskyi ***@***.***>
Date: Tuesday, 31 October 2023, 11:11
To: ita-social-projects/GreenCity ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [ita-social-projects/GreenCity] How run app (ASURE_CONECTION_STRING) (Discussion #6644)

Hello, I want to take part in the development of a project to gain experience.
Trying to run the program. I created a database locally, licibfsk created tables. But I get an error at startup because I don't have a value for AZURE_CONNECTION_STRING.
Can you tell me where to get it?
Do I need to create my own Azure Blob Storage?
